modules/microshift-audit-logs-config-intro.adoc

```diff
@@ -20,11 +20,18 @@ You can set fields in combination to define a maximum storage limit for retained
 
 {microshift-short} includes the following default audit log rotation values:
 
-* `maxFileSize`: 200Mb
-* `maxFiles`: 10 files
-* `maxFileAge`: 0, This value means that no default age limit is set.
-* `profile`: Default, This profile logs only metadata for read and write requests.
+.{microshift-short} default audit log values
+[cols="20%,20%,50%",options="header"]
+|===
 
-The maximum default storage usage for audit log retention is 2000Mb, provided that there are 10 or fewer files.
+|Audit log parameter|Default setting|Definition
+|`maxFileAge`:|`0`|How long log files are kept before automatic deletion. The default value means that a log file is never deleted based on age. This value can be configured.
+|`maxFiles`:|`10`|The total number of log files kept. By default, {microshift-short} retains 10 log files. The oldest is deleted when an excess file is created. This value can be configured.
+|`maxFileSize`:|`200`|In megabytes. By default, when the `audit.log` file reaches the `maxFileSize` limit, the `audit.log` file is rotated and {microshift-short} begins writing to a new `audit.log` file. This value can be configured.
+|`profile`:|`Default`|Logs only metadata for read and write requests; does not log request bodies except for OAuth access token requests. If you do not specify this field, the `Default` profile is used.
```

Comment:
The word _Logs_ here confused me; at first I thought it was a noun.

```suggestion
|`profile`:|`Default`|Only metadata is logged for read and write requests; does not log request bodies except for OAuth access token requests. If you do not specify this field, the `Default` profile is used.
```

Are there other profiles that can be used?

Reply:
Yes, we reference those here: https://76743--ocpdocs-pr.netlify.app/microshift/latest/microshift_configuring/microshift-audit-logs-config.html#about-audit-log-profiles_microshift-audit-logs-config. It's before the config module in the assembly.